requirements.txt

```diff
@@ -92,3 +92,7 @@ websockets==8.0.2 \
     --hash=sha256:f5cb2683367e32da6a256b60929a3af9c29c212b5091cf5bace9358d03011bf5 \
     --hash=sha256:049e694abe33f8a1d99969fee7bfc0ae6761f7fd5f297c58ea933b27dd6805f2 \
     --hash=sha256:882a7266fa867a2ebb2c0baaa0f9159cabf131cf18c1b4270d79ad42f9208dc5
+
+html2text~=2020.1.16
+requests~=2.24.0
+dateparser~=1.0.0
```

Comment:
Not a maintainer, but I used your version as a temporary work around. There was an issue in the requirements.txt file you pushed. There's a flag that requires you include the hash. Also you probably should order them alphabetically in the file.

```
html2text~=2020.1.16 \
    --hash=sha256=c7c629882da0cf377d66f073329ccf34a12ed2adf0169b9285ae4e63ef54c82b 
```

```
dateparser~=1.0.0 \
    --hash=sha256=17202df32c7a36e773136ff353aa3767e987f8b3e27374c39fd21a30a803d6f8 
```

Reply:
This file can be generated with poetry, if you don't use poetry I can update it after merging.